scipy/special/_round.h

```diff
@@ -6,15 +6,14 @@
 #define ROUND_H
 
 #include <numpy/npy_math.h>
-#include "_c99compat.h"
 #include "cephes/dd_idefs.h"
 
 
 double add_round_up(double a, double b)
 {
     double s, err;
 
-    if (sc_isnan(a) || sc_isnan(b)) {
+    if (isnan(a) || isnan(b)) {
 	return NPY_NAN;
```

Comment:
Would it make sense to get rid of these numpy nans as well here?

Reply:
Yes, I can do that - leads to a few more cleanups in `wright.cc` as well.